src/Dash.NET.Giraffe/DashApp.fs

```diff
@@ -213,34 +211,34 @@ type DashApp =
 
         //TODO: make this customizable
         let errorHandler (ex : Exception) (logger : ILogger) =
-            logger.LogError(ex, "An unhandled exception has occurred while executing the request.")
-            clearResponse >=> setStatusCode 500 >=> (loadedApp.Config.ErrorHandler ex)
-        
+           logger.LogError(ex, "An unhandled exception has occurred while executing the request.")
+           clearResponse >=> setStatusCode 500 >=> (text ex.Message)
+
         let configureCors (builder : CorsPolicyBuilder) =
-            builder.WithOrigins(sprintf "http://%s:8080" loadedApp.Config.HostName)
+            builder.WithOrigins(sprintf "http://%s:5001" args.[0])
```

Comment:
I guess it is a quick workaround now, but this needs to be properly typed so that we can create DUs for Giraffe/Suave specific configs. Binding the first entry in an arbitrary string array is very confusing, at least it was for me when testing. I would suggest to add overloads for DashApp.run in Both Dash.NET.Giraffe and Dash.NET.Suave like this:

```fsharp
type DashSuaveConfig = { Port: int ; ...}
type DashGiraffeConfig = { Port: int ; ...}

type DashApp with

 static member run (args: string []) (gConfig: DashGiraffeConfig ) (dApp: DashApp) = ...
 static member run (args: string []) (sConfig: DashSuaveConfig ) (dApp: DashApp) = ...

```